app/uk/gov/hmrc/gform/fileupload/FileUploadService.scala

```diff
@@ -77,7 +77,7 @@ class FileUploadService(
     }
 
     def uploadRoboticsXmlF: Future[Unit] = summaries.roboticsXml match {
-      case Some(elem) => uploadXml(roboticsXml, s"${submissionRef.value}-$date-robotic.xml", elem)
+      case Some(elem) => uploadXml(roboticsXml, s"$fileNamePrefix-robotic.xml", elem)
```

Comment:
Is removal of `-$date` part intentional?

Reply:
Yes, `fileNamePrefix = s"${submissionRef.withoutHyphens}-$date"`